README.md

```diff
@@ -8,13 +8,73 @@
 + Delete a key-value pair from an object
 + Iterate over key-value pairs in an object
 
-
 ## What the Student Already Knows
 
 - Variables
 - Data Type- Strings and Integers
 - Functions
-- Booleans, Comparison Operators, Flow Control 
+- Booleans, Comparison Operators, Flow Control
 - Arrays
 
+##Understanding Objects
+
+JavaScript objects are a set of key-value pairs. Objects use the JavaScript object notation, for example, `{ 'key': 'value' }`. Each key inside an object may have any value type. For example, strings, integers, booleans, and functions. JavaScript does not care what the value is as long as it maps to a key in the set.
+
```

Comment:
instead of diving right into the topic, we aim to start with what students already know or use a real world example. this provides context, engages them, and most importantly gives them the "why?" Why do objects matter? 

Here for example is how we cover hashes in Ruby- https://github.com/learn-co-curriculum/hash-overview-readme

Moreover when we start off with "objects are a set of key-value pairs," we're introducing words that the student has no idea about. "key-value" pairs mean nothing right now to a students.